### ISSUE TITLE
The output of the nowplaying api is broken - after update

### ISSUE THREAD

**Issue by kemecs** [AzuraCast/AzuraCast#5698]
### Installation Method

Docker Installation

### AzuraCast Release Channel

Rolling Release Channel

### Current AzuraCast Version

Rolling Release #[39c0372] (2022-10-09 2:24) • Docker • PHP 8.1

### What happened?

Since the big update, the nowplaying api gives meaningless answers:

curl -kX GET "https://stream.folkradio.hu/api/nowplaying" -H "accept: application/json" -H "X-API-Key: 4cb7********:778************"

Liquidsoap my Custom configuration:
folkradio24musor = audio_to_stereo(playlist(mode="normal",reload_mode="watch","/var/azuracast/stations/stream.folkradio.hu/media/musor.pls")))))

radio = switch(id="stream.folkradio.hu_schedule_switch", track_sensitive=false, [ ({ ({ ({ 0h00m01s-23h59m58s }, folkradio24musor), ({true}, radio) ])

### Relevant log output

```shell
curl -kX GET "https://stream.folkradio.hu/api/nowplaying" -H "accept: application/json" -H "X-API-Key: 4cb7bc**********4:778c57e1***************"

{"station":{"id":1,"name":"stream.folkradio.hu","shortcode":"stream.folkradio.hu","description":"Folkradio is a not-for-profit, radio service featuring traditional folk music. It primarily broadcasts Hungarian folk music, but also music of other peoples and ethnic groups living in the Carpathian Basin. ","frontend":"icecast","backend":"liquidsoap","listen_url":"http://stream.folkradio.hu:8000/folkradio.mp3","url":"https://folkradio.hu","public_player_url":"http://stream.folkradio.hu/public/stream.folkradio.hu","playlist_pls_url":"http://stream.folkradio.hu/public/stream.folkradio.hu/playlist.pls","playlist_m3u_url":"http://stream.folkradio.hu/public/stream.folkradio.hu/playlist.m3u","is_public":false,"mounts":[],"remotes":[],"hls_enabled":false,"hls_url":null,"hls_listeners":0},"listeners":{"total":11,"unique":10,"current":11},"live":{"is_live":false,"streamer_name":"","broadcast_start":null,"art":null},"now_playing":{"sh_id":78843,"played_at":1661588497,"duration":0,"playlist":"","streamer":"","is_request":false,"song":{"id":"ab574fe34f592faf5c3f27dbc52088a4","text":"AzuraCast.com - AzuraCast is Live!","artist":"AzuraCast.com","title":"AzuraCast is Live!","album":"","genre":"","isrc":"","lyrics":"","art":"https://folkradio.blob.core.windows.net/media/pages/1/58095/m3s_Folkradio_emblema_RGB_512x512_300dpi.png","custom_fields":[]},"elapsed":701,"remaining":0},"playing_next":null,"song_history":[{"sh_id":78842,"played_at":1661587601,"duration":0,"playlist":"","streamer":"","is_request":false,"song":{"id":"ab574fe34f592faf5c3f27dbc52088a4","text":"AzuraCast.com - AzuraCast is Live!","artist":"AzuraCast.com","title":"AzuraCast is Live!","album":"","genre":"","isrc":"","lyrics":"","art":"https://folkradio.blob.core.windows.net/media/pages/1/58095/m3s_Folkradio_emblema_RGB_512x512_300dpi.png","custom_fields":[]}},{"sh_id":78841,"played_at":1661587476,"duration":0,"playlist":"","streamer":"","is_request":false,"song":{"id":"ab574fe34f592faf5c3f27dbc52088a4","text":"AzuraCast.com - AzuraCast is Live!","artist":"AzuraCast.com","title":"AzuraCast is Live!","album":"","genre":"","isrc":"","lyrics":"","art":"https://folkradio.blob.core.windows.net/media/pages/1/58095/m3s_Folkradio_emblema_RGB_512x512_300dpi.png","custom_fields":[]}},{"sh_id":78840,"played_at":1661587430,"duration":0,"playlist":"","streamer":"","is_request":false,"song":{"id":"ab574fe34f592faf5c3f27dbc52088a4","text":"AzuraCast.com - AzuraCast is Live!","artist":"AzuraCast.com","title":"AzuraCast is Live!","album":"","genre":"","isrc":"","lyrics":"","art":"https://folkradio.blob.core.windows.net/media/pages/1/58095/m3s_Folkradio_emblema_RGB_512x512_300dpi.png","custom_fields":[]}},{"sh_id":78839,"played_at":1661587311,"duration":0,"playlist":"","streamer":"","is_request":false,"song":{"id":"ab574fe34f592faf5c3f27dbc52088a4","text":"AzuraCast.com - AzuraCast is Live!","artist":"AzuraCast.com","title":"AzuraCast is Live!","album":"","genre":"","isrc":"","lyrics":"","art":"https://folkradio.blob.core.windows.net/media/pages/1/58095/m3s_Folkradio_emblema_RGB_512x512_300dpi.png","custom_fields":[]}},{"sh_id":78838,"played_at":1661032803,"duration":28,"playlist":"","streamer":"","is_request":false,"song":{"id":"3ab60cf33b685676e46faa6b16d71e9c","text":"00754f1 - 00754f1","artist":"00754f1","title":"00754f1","album":"00754f1","genre":"Folklore","isrc":"","lyrics":"","art":"https://folkradio.blob.core.windows.net/media/pages/1/58095/m3s_Folkradio_emblema_RGB_512x512_300dpi.png","custom_fields":[]}}],"is_online":true,"cache":null}#
```

**Comment by kemecs** [comment]
We can change the api/nowplaying/1 url on our website, but a meaningless api response will unfortunately ruin our show display.
So if you can't fix it, I'll have to install an old version from scratch which still has the nowplaying api response.
That would be a pain for me.
Please help me!

**Comment by ItsMitchh** [comment]
Can you give us some details on your station settings, namely, please be very descriptive so we can look into this. 
Relays? 
Playlist settings? 
Are you using remote URLs?

**Comment by kemecs** [comment]
The "API" gives some kind of answer.
But not what the station is playing!

So, of course, Access-Control Allowed-Origin* doesn't help.

**Comment by ItsMitchh** [comment]
I'm not sure what you mean by key, but I'll be available in the AzuraCast Discord if you want additional help. 

https://discord.gg/azuracast

**Comment by kemecs** [comment]
I get this back, even though it's not what the server is playing:

"now_playing":{"sh_id":78843,"played_at":1661588497,"duration":0,"playlist":"","streamer":"","is_request":false,"song":{"id":"ab574fe34f592faf5c3f27dbc52088a4","text":"AzuraCast.com - AzuraCast is Live!","artist":"AzuraCast.com","title":"AzuraCast is Live!","album":"","genre":"","isrc":"","lyrics":"","art":"https://folkradio.blob.core.windows.net/media/pages/1/58095/m3s_Folkradio_emblema_RGB_512x512_300dpi.png","custom_fields":[]},"elapsed":16433,"remaining":0},

**Comment by kemecs** [comment]
Interestingly, the API response is good at the very end, but not as a whole:


{"sh_id":78838,"played_at":1661032803,"duration":28,"playlist":"","streamer":"","is_request":false,"song":{"id":"3ab60cf33b685676e46faa6b16d71e9c","text": "00754f1 - 00754f1","artist":"00754f1","title":"00754f1","album":"00754f1","genre":"Folklore","isrc":"","lyrics":""

**Comment by kemecs** [comment]
Isn't that the problem?
I see a lot of this in the Azuracast aplication log:

[2022-08-28T08:12:16.853134+00:00] AzuraCast.NOTICE: Cannot build queue: station does not support AutoDJ queue. [] {"station":{"id":1,"name":"stream.folkradio.hu"}}
[2022-08-28T08:12:27.887700+00:00] AzuraCast.NOTICE: Cannot build queue: station does not support AutoDJ queue. [] {"station":{"id":1,"name":"stream.folkradio.hu"}}
[2022-08-28T08:12:39.401841+00:00] AzuraCast.NOTICE: Cannot build queue: station does not support AutoDJ queue. [] {"station":{"id":1,"name":"stream.folkradio.hu"}}
[2022-08-28T08:12:47.568102+00:00] AzuraCast.NOTICE: Cannot build queue: station does not support AutoDJ queue. [] {"station":{"id":1,"name":"stream.folkradio.hu"}}
[2022-08-28T08:12:48.135515+00:00] AzuraCast.NOTICE: Webhook "Folkradio_google_code" has run less than 10 seconds ago; skipping... [] []
[2022-08-28T08:12:58.777670+00:00] AzuraCast.NOTICE: Cannot build queue: station does not support AutoDJ queue. [] {"station":{"id":1,"name":"stream.folkradio.hu"}}
[2022-08-28T08:13:15.534551+00:00] AzuraCast.NOTICE: Cannot build queue: station does not support AutoDJ queue. [] {"station":{"id":1,"name":"stream.folkradio.hu"}}

**Comment by kemecs** [comment]
...and
Liquidsoap Log has two oddities at startup:

2022/08/28 10:37:54 [lang.deprecated:2] WARNING: "map_metadata" is deprecated and will be removed in future version. Please use "metadata.map" instead.
----
....
2022/08/28 10:37:54 [lang:3] API feedback - Sending POST request to 'http://127.0.0.1:6010/api/internal/1/liquidsoap/feedback' with body: { "artist": "AzuraCast.com", "title": "AzuraCast is Live!" }

**Comment by kemecs** [comment]
There may be a connection:
https://github.com/AzuraCast/AzuraCast/discussions/5670

**Comment by kemecs** [comment]
...how can I install an earlier version of azurecast?
It is desperate that there is no way to do it using the docker method, if I read the notes correctly.

I would have to somehow revert to the multi-container version to get the nowplaying api working again..

**Comment by kemecs** [comment]
I use Liquidoap in the Custom setting:

folkradio24musor = 
audio_to_stereo(playlist(mode="normal",reload_mode="watch","/var/azuracast/stations/stream.folkradio.hu/media/musor.pls"))
radio = switch(id="stream.folkradio.hu_schedule_switch", track_sensitive=false, [ ({ 0h00m01s-23h59m58s }, folkradio24musor), ({true}, radio) ])

I have now turned it off and created a playlist with azuracast.

Then the nowplaying api starts to respond properly!

This is the problem. Because we are using a custom, not random playlist!

So we need a way to dowgrade or install an old version that we can still work with...

**Comment by ItsMitchh** [comment]
We don't give support or assistance on downgrading, have patience

**Comment by ItsMitchh** [comment]
We don't give support or assistance on downgrading, have patience

**Comment by kemecs** [comment]
I've been testing a lot lately...
(all playlist formats M3U or PLS)

Most recently with this custom setting:
folkradio24musor =
audio_to_stereo(playlist(mode="normal",reload_mode="watch","/var/azuracast/stations/stream.folkradio.hu/media/musor.m3u"))
radio = switch(id="stream.folkradio.hu_schedule_switch", track_sensitive=false, [ ({ 0h00m01s-23h59m58s }, folkradio24musor), ({true}, radio) ])

In all cases I find that the nowplaying api response is wrong with the Rolling Release Channel, and also with the Stable version.

Either the output of "playing_next" or "now_playing" is wrong!

"now_playing":{"sh_id":0,"played_at":0,"duration":0,"playlist":null,"streamer":null,"is_request":false,"song":{"id":"5a6a865199cf5df73b1417326d2ff24f","text":" - Station Offline"

I see a lot of this in the log:

2022/09/25 15:44:55 [lang:3] API nextsong - Send POST request to 'http://127.0.0.1:6010/api/internal/1/liquidsoap/nextsong' with body:
2022/09/25 15:44:55 [lang:3] API nextsong - Response (200): false

(This is a problem because our website shows what we are currently playing based on a "API" query.)


Is there any chance that this can be fixed somehow? Or should I try another way to start the sequential playlist, in Custom mode? 
(I tried to switch to "Remote URL Playlist", but there I ran into the sequential playlist problem.)


_I really like the AzuraCast system and appreciate your work!!_
